packages/js/package.json

```diff
@@ -19,7 +19,9 @@
   },
   "dependencies": {
     "@silverhand/essentials": "^1.1.2",
-    "js-base64": "^3.7.2"
+    "jose": "^4.1.4",
```

Comment:
this should be put in `devDependencies` for now, or users will install `jose` while they don't need

Reply:
My bad.

But it will be required by `verifyIdToken` in the next PR.
I prefer keep it for now.